From: Michael Biebl <biebl@debian.org>
Date: Thu, 18 Jul 2013 15:33:51 +0200
Subject: Add note to udev.conf that changes to that file require a rebuild of
 the initramfs

Based on debian/patches/udev_conf_comments from the old udev package.
---
 src/udev/udev.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/udev/udev.conf b/src/udev/udev.conf
index 47d1433..b7a42ab 100644
--- a/src/udev/udev.conf
+++ b/src/udev/udev.conf
@@ -1,3 +1,6 @@
 # see udev.conf(5) for details
+#
+# udevd is started in the initramfs, so when this file is modified the
+# initramfs should be rebuilt.
 
 #udev_log="info"
